scripts/eols.sh

```diff
@@ -11,7 +11,7 @@ SELECT
   r.major,
   r.minor,
   r.date,
-  (SELECT DATE(MIN(r2.date), '+1 year')
```

Comment:
This changes the output for release metadata so that the EOL dates now are in RFC3339 format. The only upstream consumer right now is Docs site, which is doing a [`new Date(d)`](https://github.com/launchdarkly/ld-docs-private/blob/main/src/components/sdksEndOfLife.tsx#L75), so should continue to work.